src/main/java/yoda/ui/Ui.java

```diff
@@ -0,0 +1,104 @@
+package yoda.ui;
+
+import yoda.task.Task;
+
+/**
+ * Ui class to handle interactions with the user.
+ */
+public class Ui {
+    /** Valid inputs that are available to the user */
+    private static final String HELP_LIST = "The commands you can use, these are:\n"
+                                            + "To add a task:\n"
+                                            + "todo [description]\n"
+                                            + "event [description] /at YYYY-MM-DD HHmm\n"
+                                            + "deadline [description] /by YYYY-MM-DD HHmm\n"
+                                            + "To list tasks:\n"
+                                            + "list (-t/-e/d)\n"
+                                            + "To find all tasks related to a keyword:\n"
+                                            + "find [keyword]\n"
+                                            + "To mark a task as done:\n"
+                                            + "done [taskNumber_in_the_list]\n"
+                                            + "To delete a task:\n"
+                                            + "delete [taskNumber_in_the_list]";
```

Comment:
I'm not sure whether this is an acceptable break from indentation for wrapped lines having 8 spaces? Also, I would suggest you align the pluses with the equal sign, not with the first double quote symbol